vignettes/trial_analysis.bib

```diff
@@ -0,0 +1,13 @@
+% Encoding: UTF-8
+
+@Article{NEUENSCHWANDER,
```

Comment:
maybe it would be sufficient to just have one .bib file that is shared across vignettes?

Reply:
I thought the same, but since the JSS paper vignette had a .bib file that was clearly specific to itself, I followed the same model.  I will combine the .bib files and make the appropriate edits to the vignettes.

Reply:
thanks that will be great!

Reply:
I have updated the .bib files and vignettes as agreed.  However, `crmPack-jss-paper.Rnw` does not weave successfully.  I don't think this has anything to do with the change in bibliography.  The error is
```
Line 378: error in eval(parse_only(code), envir = envir)
```
Lines 370 to 386 are
```
370: <<min-inf, warning=FALSE, message=FALSE, results="hide">>=
371: coarseGrid <- c(25, 50, 100, 200, 300)
372: model <- MinimalInformative(dosegrid = coarseGrid, refDose = 100,
373:                             logNormal = TRUE, threshmin = 0.1,
374:                             threshmax = 0.2, seed = 432,
375:                             control = list(max.time = 30))$model
276: @
377: 
378: The resulting \code{model} (which is an object of class \code{LogisticLogNormal})
379: has prior parameters
380: $\mu = (\Sexpr{round(model@params@mean[1], 2)}, \Sexpr{round(model@params@mean[2], 2)})$ and
381: $\Sigma=
382: \left(\begin{smallmatrix}
383:   \Sexpr{round(model@params@cov[1, 1], 2)} & \Sexpr{round(model@params@cov[1, 2], 2)} \\
384:   \Sexpr{round(model@params@cov[2, 1], 2)} & \Sexpr{round(model@params@cov[2, 2], 2)}
385: \end{smallmatrix}\right)
386: $
```

@danielinteractive @TJaki @palermogp  Would one of the authors please take a look?

Reply:
could it just be change to `ref_dose` needed?